apps/web/components/booking/pages/AvailabilityPage.tsx

```diff
@@ -1,6 +1,7 @@
 // Get router variables
 import { EventType } from "@prisma/client";
 import * as Collapsible from "@radix-ui/react-collapsible";
+import DOMPurify from "dompurify";
```

Comment:
On further inspection, `dompurify` isn't needed if we use `markdown-it` in strict mode.